.changes/unreleased/tinymce-TINY-10574-2024-01-15.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Changed
+body: now `highlight_on_focus` default is true
```

This is a relatively new option, I think we still need to explain how this change will impact integrators
```suggestion
body: The `highlight_on_focus` option now defaults to true, adding a focus outline to every editor
```

---

done